pyproject.toml

```diff
@@ -36,7 +36,7 @@ python = ">=3.8,<3.11"
 [tool.poetry.dev-dependencies]
 mypy = ">=0.960"
 pytest = ">=7.1.2"
-pyright = ">=1.1.251"
+pyright = ">=1.1.255"
```

Comment:
pyright has released `1.1.257`.  We'll have to figure out a way to keep that updated

Reply:
Without a lock file, poetry will try to install the latest version anyways. If there is a cached lock file, it might use an older version.

Even if we install an old pyright (from pip, not the "real" pyright), the pip-pyright will first check whether there is a newer version of the "real" pyright. https://pypi.org/project/pyright/

> We also automatically upgrade the pyright npm package to its latest version on every run, see below for how to change this behaviour.

I just bumped the version to check the speed impact when we can use the old venv but have to re-create the lock file.